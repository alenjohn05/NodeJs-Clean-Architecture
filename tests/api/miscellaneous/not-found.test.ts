import { client } from '../client';

describe('GET /does-not-exist', () => {
  test('responds with a 404 status code', async () => {
    const response = await client.get('/does-not-exist');

    expect(response.status).toEqual(404);
    expect(response.body).toMatchSnapshot();
  });
});
