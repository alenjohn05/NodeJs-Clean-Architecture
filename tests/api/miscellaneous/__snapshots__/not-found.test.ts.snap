// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GET /does-not-exist responds with a 404 status code 1`] = `
{
  "status": 404,
  "title": "The specified resource was not found.",
  "type": "https://tools.ietf.org/html/rfc7231#section-6.5.4",
}
`;
